packages/giselle-engine/src/core/generations/utils.ts

```diff
@@ -109,7 +109,7 @@ async function buildGenerationMessageForTextGeneration(
 			case "text": {
 				userMessage = userMessage.replace(
 					replaceKeyword,
-					contextNode.content.text,
+					jsonContentToText(JSON.parse(contextNode.content.text)),
```

Comment:
It's difficult to understand because it's not clearly represented in the TypeScript types, but `contextNode.content.text` can be a string type, not just a `jsonContent` type.  Before parsing, please check if it's actually a `jsonContent` type. If it's not, please leave it as is (don't attempt to parse it).

I think the following code might be helpful as a reference:

https://github.com/giselles-ai/giselle/blob/e71f136fecaed452175c12f06ad62f48f709e089/packages/giselle-engine/src/core/generations/utils.ts#L89C1-L91C3